tests/unit/__setup.test.js

```diff
@@ -0,0 +1,16 @@
+var chai = require('chai');
+var path = require('path');
+var Sinon = require('sinon');
+
+var core = require('../../core');
+core.setNativeBindingsModule(path.resolve('native-bindings.js'));
+
+global.assert = chai.assert;
+
+beforeEach(function () {
+  this.sinon = Sinon.createSandbox();
```

Comment:
I suspect `sinon` might fight with exokit re: some of the things we'll want to test. But that's a problem for another day.

Reply:
How do you imagine it might fight?

Reply:
Things like timeouts, rafs, error traps, and request species are important for getting the web APIs right, and exokit has to jump a couple hoops to intercept and remap things.

If `sinon` is playing the same game I can imagine one attempting to overwrite the other. Just a hunch.

But... it would probably be fixable if we see it.